src/assets/text/helloworld.nf

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nextflow
+
+// tutorial.nf example from https://www.nextflow.io/docs/latest/getstarted.html
+
+params.str = 'Hello world!'
+
+process splitLetters {
+
+    output:
+    file 'chunk_*' into letters
+
+    """
+    printf '${params.str}' | split -b 6 - chunk_
+    """
+}
+
+
+process convertToUpper {
+
+    input:
```

Comment:
Example should use a pinned ubuntu Docker image

Reply:
Also need to add a sample `nextflow.config`